src/app/Scenes/MyProfile/Components/MyProfileHeader.tsx

```diff
@@ -9,7 +9,7 @@ export const MyProfileHeader: React.FC = () => {
 
   return (
     <Flex justifyContent="center" alignItems="center" mt={2}>
-      <Touchable onPress={() => navigate("my-collection")}>
+      <Touchable onPress={() => navigate("/collector-profile/my-collection")}>
```

Comment:
https://github.com/artsy/eigen/blob/66124d373b2aa4e3860f7d3d646b7dbb7cc66bf7/docs/routing.md#L1

```suggestion
      <RouterLink to="/collector-profile/my-collection">
```